heron/storm/src/java/BUILD

```diff
@@ -8,21 +10,29 @@ storm_deps_files = [
     "//third_party/java:kryo",
 ]
 
+storm_deps_files_nl = [
+    "//heron/api/src/java:api-java-neverlink",
+    "//heron/common/src/java:basics-java",
+    "//heron/simulator/src/java:simulator-java-neverlink",
```

Comment:
Do we depend on simulator to implement `LocalCluster` in heron-storm library? Or, if we removed `LocalCluster`, I recommend do not depend on `simulator` from `heron-storm`. I guess publishing of separate `testing` artifact would be useful for that.